inference/server/main.py

```diff
@@ -148,3 +148,19 @@ async def maybe_add_debug_api_keys():
 async def welcome_message():
     logger.warning("Inference server started")
     logger.warning("To stop the server, press Ctrl+C")
+
+
+if __name__ == "__main__":
+    import uvicorn
+    import os
+
+    port = int(os.getenv('PORT', "8000"))
+    is_debug = bool(os.getenv("DEBUG", "False"))
+
+    if is_debug:
+        import debugpy
+        debugpy.listen(("0.0.0.0", "5679"))
+        # Uncomment to wait here until a debugger is attached
+        # debugpy.wait_for_client()
+    
+    uvicorn.run("main:app", host="0.0.0.0", port=port, reload=is_debug)
```

Comment:
This method of starting the server is only used for development - the docker image for production still invokes the `uvicorn` command